packages/tldraw/src/lib/ui/hooks/useActions.tsx

```diff
@@ -1211,3 +1255,14 @@ export function useActions() {
 function asActions<T extends Record<string, TLUiActionItem>>(actions: T) {
 	return actions as Record<keyof typeof actions, TLUiActionItem>
 }
+
+/** @public */
+export function unwrapLabel(label?: TLUiActionItem['label'], menuType?: string) {
```

Comment:
too hard to read :)
(i overlove ternaries too so sometimes i have the same problem :P)